doc/rados/troubleshooting/memory-profiling.rst

```diff
@@ -2,16 +2,21 @@
  Memory Profiling
 ==================
 
-Ceph MON, OSD and MDS can generate heap profiles using
-``tcmalloc``. To generate heap profiles, ensure you have
-``google-perftools`` installed::
+Ceph MON, OSD and MDS can generate heap profiles using ``tcmalloc``. Make sure
```

Comment:
Ceph Monitor, OSD, and MDS daemons can report ``TCMalloc`` heap profiles.  Install ``google-perftools`` on the daemon systems if you wish to generate these.  Your OS distribution may package this under a different name, like ``gperftools``, and use a different package manager.

Reply:
Accepted with slight modifications.